src/main/java/com/minecolonies/colony/CitizenData.java

```diff
@@ -109,8 +109,8 @@ public CitizenData(int id, Colony colony)
     @NotNull
     public static CitizenData createFromNBT(@NotNull NBTTagCompound compound, Colony colony)
     {
-        int id = compound.getInteger(TAG_ID);
-        @NotNull CitizenData citizen = new CitizenData(id, colony);
+        final int id = compound.getInteger(TAG_ID);
+        final @NotNull CitizenData citizen = new CitizenData(id, colony);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)